app/tests/test_user.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import status
+
+from app.tests.base import BaseTestCase
+
+
+class UserTestCase(BaseTestCase):
+    def test_user_change_password(self):
```

Comment:
this is only positive test case, can you write negative test cases as well?

Reply:
added